tests/test_cli_reason_peer_seeds.py

```
@@ -0,0 +1,380 @@
+
+
+# ---------------------------------------------------------------------------
+# Query path: peer seeds union
+# ---------------------------------------------------------------------------
+
+
+def test_query_path_surfaces_peer_seed_human(
+    two_scope_with_local: tuple,
+) -> None:
+    """Query path: peer FTS5 hit surfaces with [scope:peerA] in human output."""
+    _local_db, _peer_db, local_id, peer_id = two_scope_with_local
```

Ruff (0.15.12)

[warning] Unpacked variable `local_id` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)